<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetricFilter</class>
 <widget class="QWidget" name="MetricFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>114</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="cmbMetric">
     <item>
      <property name="text">
       <string>&lt;None&gt;</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="frmThreshold">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Threshold</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSlider" name="sldThreshold">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnThreshold">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sldThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spnThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnThreshold</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sldThreshold</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
